syntax = "proto3";
package patch;





/*
UpdateWinPatch请求
*/
message UpdateWinPatchRequest {
    /*
    补丁ID
    */
    string patchId = 1 ;
    /*
    补丁编号
    */
    string articleId = 2 ;
    /*
    MSRC编号
    */
    string msrc = 3 ;
    /*
    发布时间
    */
    string releaseTime = 4 ;
    /*
    是否需要重启
    */
    bool requireReboot = 5 ;
    /*
    备注
    */
    string memo = 6 ;
}

/*
UpdateWinPatch返回
*/
message UpdateWinPatchResponse {
    /*
    补丁id
    */
    string instanceId = 1 ;
}

/*
UpdateWinPatchApi返回
*/
message UpdateWinPatchResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    patch.UpdateWinPatchResponse data = 4 ;
}

